tests/test_schema.py

```diff
@@ -5,7 +5,7 @@
 def test_schema_entrypoint() -> None:
     if sys.version_info < (3, 10):
         eps = importlib.metadata.entry_points()["validate_pyproject.tool_schema"]
-        (black_ep,) = [ep for ep in eps if ep.name == "black"]
+        (black_ep,) = (ep for ep in eps if ep.name == "black")
```

Comment:
I know pyupgrade does this but I don't like this change. It feels harder to understand (generator expressions are rarer than listcomps). The genexp might sometimes be faster, but it's unlikely to matter.

In this case, the new code appears to be slower for me (benchmark on Linux on 3.9):

```
In [8]: eps = importlib.metadata.entry_points()["validate_pyproject.tool_schema"]

In [9]: eps
Out[9]: (EntryPoint(name='black', value='black.schema:get_schema', group='validate_pyproject.tool_schema'),)

In [10]: %timeit (black_ep,) = [ep for ep in eps if ep.name == "black"]
170 ns ± 1.32 ns per loop (mean ± std. dev. of 7 runs, 10,000,000 loops each)

In [11]: %timeit (black_ep,) = (ep for ep in eps if ep.name == "black")
232 ns ± 1.61 ns per loop (mean ± std. dev. of 7 runs, 1,000,000 loops each)
```